src/cassandra/session.rs

```diff
@@ -135,6 +137,19 @@ impl Session {
         unsafe { <CassFuture<CassResult>>::build(cass_session_execute(self.0, statement.inner())) }
     }
 
+    /// Execute a statement and get any custom payloads.
```

Comment:
Please can you implement `execute_batch_with_payloads` as well?

Reply:
Yes, sure.